pom.xml

```diff
@@ -8,5 +8,86 @@
     <artifactId>VadimPkin</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <properties>
+        <java.source.version>1.8</java.source.version>
+        <java.target.version>1.8</java.target.version>
 
+        <testng.version>7.1.0</testng.version>
+
+        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+        <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+
+        <calculator.version>1.0</calculator.version>
+    </properties>
+
+    <profiles>
+        <profile>
+            <id>all</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <suite.xml>src/test/resources/TestSuites/All_operations_test_suite.xml</suite.xml>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>addition_and_subtract</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <suite.xml>src/test/resources/TestSuites/Addition_and_subtraction_test_suite.xml</suite.xml>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>multiply_and_division</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
Why is this profile active by default? Which profile should be active by default from the HW requirements?

Reply:
> Why is this profile active by default? Which profile should be active by default from the HW requirements?

Addition and subtraction should be. Fixed that issue.